src/app/home/panel/navbar/navbar.component.scss

```diff
@@ -46,3 +46,28 @@
   font-size: 0.6rem;
   margin-top: 0.3rem;
 }
+
+@media (max-width: 359px) {
+  .element-container {
+    margin: 0 .4rem 0 .4rem;
+  }
+}
+
+@media (min-width: 360px) {
+  .element-container {
+    margin: 0 .7rem 0 .7rem;
+  }
+}
+
+@media (min-width: 376px) {
+  .element-container {
+    margin: 0 1rem 0 1rem;
+  }
+}
+
+@media (min-width: 391px) {
+  .element-container {
+    margin: 0 1.2rem 0 1.2rem;
+  }
+}
```

Comment:
Do you really need that many different media queries? Those are so close. I guess two shoud be fine.